stacks-signer/src/runloop.rs

```diff
@@ -1,8 +1,8 @@
-use hashbrown::{HashMap, HashSet};
 use std::collections::VecDeque;
 use std::sync::mpsc::Sender;
 use std::time::Duration;
 
+use hashbrown::{HashMap, HashSet};
```

Comment:
Is there a reason we're still using `hashbrown`?

Reply:
Unclear to me. Maybe open an issue for the signer? This changeline just came from running `cargo fmt-stacks`.